assets/i18n/pt-br.json

```diff
@@ -192,6 +192,7 @@
   "I18N_PLAYER_RETURN_TO_EDITOR": "Voltar ao editor",
   "I18N_PLAYER_RETURN_TO_LIBRARY": "Voltar à biblioteca",
   "I18N_PLAYER_SHARE_EXPLORATION": "Gostou desta exploração? Compartilhe com seus amigos!",
+  "I18N_PLAYER_SHARE_THIS_EXPLORATION" : "",
```

Comment:
Added. Wonder why this wasnt caught in the i18n checks of testing? Just curious.